.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Pull Request template
+Please, go through these steps before you submit a PR.
+
+1. This repository follows semantic versioning convention, therefore each PR title/commit message must follow convention: `<type>(<scope>): <subject>`.
+   `type` is defining if release will be triggering after merging submitted changes, details in [CONTRIBUTING.md](../CONTRIBUTING.md).
+    Most common types are:
+    * `feat`      - for new features, not a new feature for build script
```

Comment:
`not a new feature for build script`, 
@mynktl How should PRs affecting build script be formatted?

Reply:
using `chore`